config/schemas/terrain.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Have not got any schema for this so far? That is strange.

Reply:
yes, missed that:(